tests/acceptance/bootstrap/SettingsContext.php

```diff
@@ -497,4 +497,56 @@ public function theUserHasDisabledAutoAccepting(string $user): void {
 		);
 		$this->featureContext->rememberUserAutoSyncSetting($user, false);
 	}
+
+	/**
+	 * @When user :user disables email notification using the settings API
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 */
+	public function userDisablesEmailNotificationUsingTheSettingsAPI(string $user): void {
+		$response = HttpRequestHelper::sendRequest(
+			$this->featureContext->getBaseUrl() . "/api/v0/settings/values-save",
+			$this->featureContext->getStepLineRef(),
+			'POST',
+			$this->featureContext->getActualUsername($user),
+			$this->featureContext->getPasswordForUser($user),
+			null,
+			json_encode($this->createBodyForNotificationSetting($user)),
+		);
+		$this->featureContext->setResponse($response);
+	}
+
+	/**
+	 * @param string $user
+	 *
+	 * @return array
+	 */
+	public function createBodyForNotificationSetting(string $user): array {
```

Comment:
```suggestion
	public function createBodyForNotificationSetting(string $user): array {
```
how about `getBodyForNotificationSetting` or `getPayloadForNotificationSetting` ?